src/Dependencies/CodeAnalysis/project.json

```diff
@@ -1,6 +1,10 @@
 ﻿{
   "dependencies": {
-    "Microsoft.CodeAnalysis": "1.2"
+    "Microsoft.CodeAnalysis": "1.2",
+    "FakeSign": "0.9.2",
```

Comment:
Ideally we'd want another project called Toolset that everything depended on because it's conceivable that there would be projects which require the toolset but not codeanalysis.but since that's not the case today maybe this is okay - @davkean?